mmv1/third_party/terraform/services/cloudidentity/resource_cloud_identity_group_test.go.erb

```diff
@@ -104,6 +104,10 @@ func testAccCloudIdentityGroup_cloudIdentityGroupsBasicExampleTest(t *testing.T)
 		Steps: []resource.TestStep{
 			{
 				Config: testAccCloudIdentityGroup_cloudIdentityGroupsBasicExample(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("google_cloud_identity_group.cloud_identity_group_basic",
```

Comment:
Could you clarify what is happening here? I would expect to be able to rely on the `ImportStateVerify` (and if any fields are settable, I would expect to see them added to an example).

Reply:
I intended that this line would check that the `additional_group_keys` field is set, without asserting anything about its contents or length. I could simplify it to remove the `.#` as I think that stores information about the lists length ([see example here](https://github.com/GoogleCloudPlatform/magic-modules/blob/4dc1173c6943b348308001f9071b37581aa75e01/mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.erb#L1540-L1557)) and using just `"additional_group_keys"`would be less confusing. 

I wasn't really sure how I should update a test to reflect that an attribute has been added successfully. `ImportStateVerify` would check that the imported resource matches the provisioned resource, but wouldn't assert anything about an attribute being present or not. I could even remove this addition to the tests as it's maybe being a bit over-cautious.

What do you think would be appropriate?

Reply:
I found https://github.com/hashicorp/terraform-provider-google/wiki/Developer-Best-Practices#test-checks, which basically supports what you've done here. I don't see this used often, even though we have a lot of computed fields, so perhaps it is on the cautious side, but IMO it's reasonable to add for this PR. Thanks for explaining!